README.md

```diff
@@ -289,6 +289,24 @@ You can ask Spago to come up with a good set of bounds for you by running:
 $ spago install --ensure-ranges
 ```
 
+You can specify your version ranges manually in the `spago.yaml` configuration file too:
+
+```yaml
+package:
+  dependencies:
+    - lists: ">=7.0.0 <8.0.0"
+```
+
+If spago cannot find package versions within the specified ranges it will warn you on install:
+
+```console
+‼ The following packages versions do not exist in your package set:
+  - lists: >=1000.0.0 <1000.0.1
+
+Proceeding with the latest available versions instead:
+  - lists: 7.0.0
+```
+
```

Comment:
I'd like to remove this part since we normally do not report expected error messages in the docs, as it should be the other way around: error messages should point at the docs whenever there's more clarification needed and to show how to do things correctly (and this is not the case since the error message is well explained)
```suggestion
```